xtask/codegen/src/generate_crate.rs

```diff
@@ -0,0 +1,50 @@
+use std::fs;
+use xtask::*;
+
+fn cargo_template(name: &str) -> String {
+    format!(
+        r#"
+[package]
+authors.workspace    = true
+categories.workspace = true
+description          = "<DESCRIPTION>"
+edition.workspace    = true
+homepage.workspace   = true
+keywords.workspace   = true
+license.workspace    = true
+name                 = "{name}"
+repository.workspace = true
+version              = "0.0.0"
+[lints]
+workspace = true    
+"#
+    )
+}
+
+fn knope_template(name: &str) -> String {
+    format!(
+        r#"
+[package."{name}"]
```

Comment:
I got an error when execute `just new-changeset` after adding new create.
```suggestion
[packages.{name}]
```